index.bs

```diff
@@ -1656,9 +1682,11 @@ counterparts for default controllers, as discussed in [[#rs-abstract-ops-used-by
  1. If |autoAllocateChunkSize| is not undefined,
   1. Let |buffer| be [$Construct$]({{%ArrayBuffer%}}, « |autoAllocateChunkSize| »).
```

Comment:
Unrelated to this PR, but it looks like we're missing the *"...and interpreting the result as a completion record"* here.

Reply:
Well, generally we always treat the result as a completion record, unless it's unwrapped with ! or ?

Reply:
Oh woops, you're right. I missed that.